hydra/_internal/utils.py

```diff
@@ -300,10 +302,17 @@ def get_args_parser() -> argparse.ArgumentParser:
 
     Uninstall:
     {}
+
+    Fish shell install:
+    {}
+
+    Fish shell uninstall:
+    {}
 """.format(
-            install_cmd, uninstall_cmd
+            install_cmd, uninstall_cmd, fish_install_cmd, fish_uninstall_cmd
         ),
     )
+
```

Comment:
Please clear this from the diff and create a new issue to improve the completion plugin interface to support per-plugin help.

Reply:
I'm not sure I know what you mean by clearing it without modifying the interface. I can extend that help interface as part of this pr.

Reply:
That works too.
How 
```python
class CompletionPlugin(Plugin):

    @abstractmethod
    def help(self, command: str):
        """
        :param command: "install" or "uninstall"
        :return: command the user can run to install or uninstall this shell completion on the appropriate shell
        """
        ...

```

Reply:
Cool.